classes/core-image-pelux.bbclass

```diff
@@ -4,26 +4,37 @@
 #
 
 inherit core-image
+inherit extrausers
+
+IMAGE_FEATURES += "ssh-server-openssh package-management"
```

Comment:
Does that build for you ? Especially the package-management didn't build for me without also adding IMAGE_FEATURES += ${DISTRO_FEATURES_LIBC}

Reply:
It would be nice if we make sure to use the same ssh-server as we do in dev images which I belive is drop-bear?

Reply:
@Gagi2k yes, this is not a change from before actually (just moving the lines around). Also the poky include sets `DISTRO_FEATURES ?= "... ${DISTRO_FEATURES_LIBC} ..."`

@DunderRoffe I wouldn't mind that at all actually. Either to use dropbear in non-dev or to use openssh in -dev images.

Reply:
The non-debug image shouldn't contain an SSH server at all, right? Since you won't ever log in to it via SSH...

Reply:
No, that makes sense. I'll remove ssh from the `IMAGE_FEATURES` for non-dev.